### DISCUSSION TITLE
nvDpyUpdateHdmiVRRCaps function

### DISCUSSION THREAD

**Discussion by kahveciderin** [NVIDIA/open-gpu-kernel-modules#176] [Q&A]
this function has an empty body, and seems to be related to hdmi based on its name. Is this just a placeholder for proprietary code or is this how it's meant to be?

**Comment by ElijahPepe** [comment]
I assume the function is designed to parse the variable refresh rate on some monitors that support VRR and are running through HDMI.

I'm not entirely sure why the function is blank. Parsing the VRR on HDMI 2.1 displays shouldn't be proprietary and IIRC eARC is supported anyway.

**Reply by aritger** [reply]
For a variety of reasons, we weren't able to include G-Sync/VRR support in the initial release of open-gpu-kernel-modules.  When we finally enable G-Sync/VRR for open-gpu-kernel-modules, this function, and other related VRR functions, will be fleshed out.
